posts/2018-11-20.md

```diff
@@ -162,7 +162,7 @@ let horror: Collection<_, [String; 6]>;
 and what we want most is to assign unique identifiers to each six-tuple.
 
 ```rust
-// Use the string interning method.
+// Use the string interning method up above.
 let horror_ids = intern(&horror);
```

Comment:
looks like you changed the function name, without updating the call here. Correct?